cocotb/bfmgen.py

```diff
@@ -0,0 +1,364 @@
+###############################################################################
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+###############################################################################
```

Comment:
Please add a docstring describing this module below this line. Even a brief description is better than nothing.

Reply:
Done.